src/plotting/task_plot_incidences.py

```diff
@@ -3,18 +3,53 @@
 import pytask
 
 from src.config import BLD
+from src.config import SRC
 from src.plotting.plotting import plot_incidences
+from src.policies.policy_tools import filter_dictionary
 from src.simulation.task_process_simulation_outputs import (
     create_path_for_weekly_outcome_of_scenario,
 )
-from src.simulation.task_process_simulation_outputs import OUTCOMES
 from src.simulation.task_run_simulation import FAST_FLAG
 from src.simulation.task_run_simulation import NAMED_SCENARIOS
 
+PY_DEPENDENCIES = {
+    "py_config": SRC / "config.py",
+    "py_plot_incidences": SRC / "plotting" / "plotting.py",
+    "py_process_sim_outputs": SRC / "simulation" / "task_process_simulation_outputs.py",
+}
```

Comment:
Used multiple times. maybe to plotting/__init__.py and then import from there. I am sure the task file is not necessary.

Reply:
Thanks for catching both. I moved the `PY_DEPENDENCIES` to `plotting.py` and import from there.